Services/ImageService.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using ReleaseNotes_WebAPI.Domain.Services;
+using ReleaseNotes_WebAPI.Domain.Services.Communication;
+
+namespace ReleaseNotes_WebAPI.Services
+{
+    public class ImageService : IImageService
+    {
+        private readonly string _imagesDirectory = "images";
+        private readonly string _storedImagesPath;
+        private readonly string _baseUrl;
+
+        public ImageService(IWebHostEnvironment environment, IHttpContextAccessor httpContext)
+        {
+            _baseUrl = Path.Combine("http://" + httpContext.HttpContext.Request.Host.Value, _imagesDirectory);
+            _storedImagesPath = Path.Combine(environment.WebRootPath, _imagesDirectory);
+        }
+
+        public async Task<ImageResponse> SaveToFilesystem(IFormFile file)
+        {
+            if (file.Length <= 0) return new ImageResponse(false, "Image cannot be empty");
+
+            var fileName = Guid.NewGuid() + Path.GetExtension(file.FileName);
+            var filePath = Path.Combine(_storedImagesPath, fileName);
+            await using (var stream = File.Create(filePath))
+            {
+                await file.CopyToAsync(stream);
+            }
```

Comment:
E du sekker på at det bi generert et unikt navn? 

I det tifelle at man start applikasjonsserveren på nytt, vil den da husk ka som e søtte?

Reply:
Ja, dinna metoden e ganske sikker, ser ut som id'en blir generert av OSet

https://docs.microsoft.com/en-us/dotnet/api/system.guid?view=netframework-4.8

Reply:
https://stackoverflow.com/questions/39771/is-a-guid-unique-100-of-the-time

Reply:
Ser ut som at det e sannsynligheten for to like e såpass liten at det e mulig å bruk den der, men går an å lag error handling på det, regn med den File.create()-metoden hiv en exception på samme navn eller nå